fedmsg_meta_umb/clair.py

```diff
@@ -20,82 +20,92 @@
 
 
 class ClairProcessor(BaseProcessor):
-    topic_prefix_re = r'/topic/VirtualTopic\.eng'
+    topic_prefix_re = r"/topic/VirtualTopic\.eng"
 
-    __name__ = 'clair'
-    __description__ = 'a service responsible for container grading'
-    __link__ = 'http://clair-clair-prod.cloud.paas.psi.redhat.com'
-    __docs__ = 'https://docs.engineering.redhat.com/x/OhbhB'
-    __icon__ = '_static/img/icons/clair.png'
-    __obj__ = 'Clair'
+    __name__ = "clair"
+    __description__ = "a service responsible for container grading"
+    __link__ = "http://clair-clair-prod.cloud.paas.psi.redhat.com"
+    __docs__ = "https://docs.engineering.redhat.com/x/OhbhB"
+    __icon__ = "_static/img/icons/clair.png"
+    __obj__ = "Clair"
 
     def title(self, msg, **config):
-        return msg['topic'].split('.', 2)[-1]
+        return msg["topic"].split(".", 2)[-1]
 
     def subtitle(self, msg, **config):
-        inner_msg = msg['msg']
-        topic = msg['topic']
+        inner_msg = msg["msg"]
+        topic = msg["topic"]
 
         if not isinstance(inner_msg, dict):
             return "Unknown message format"
 
-        template = 'Unknown message'
+        template = "Unknown message"
 
-        if topic.endswith('clair.notification'):
-            if inner_msg.get('affected_images'):
-                inner_msg['affected_images_len'] = len(inner_msg['affected_images'])
+        if topic.endswith("clair.notification"):
+            if inner_msg.get("affected_images"):
+                inner_msg["affected_images_len"] = len(inner_msg["affected_images"])
                 template = self._(
-                    'New notification batch {vulnerability_name} with {affected_images_len} image(s)'
+                    "New notification batch {vulnerability_name} with {affected_images_len} image(s)"
                 )
             else:
-                template = self._('New notification {Notification[Name]}')
-        elif topic.endswith('clair.scan'):
-            template = self._('New image scanned: {nvr}.{arch} ({grades[0][grade]})')
-        elif topic.endswith('clair.scan.isv'):
-            template = self._('New image scanned: {image_id} ({grades[0][grade]})')
-        elif topic.endswith('clair.command'):
-            if inner_msg.get('brew_build'):
-                template = self._('Scan request: {brew_build}.{architecture}')
-            elif inner_msg.get('imageDigest'):
-                template = self._('Scan request: {imageDigest}')
+                template = self._("New notification {Notification[Name]}")
+        elif topic.endswith("clair.scan"):
+            if inner_msg.get("nvr") and inner_msg.get("arch"):
+                template = self._(
+                    "New image scanned: {nvr}.{arch} ({grades[0][grade]})"
+                )
+            elif inner_msg.get("image_id"):
```

Comment:
In what cases is `nvra` present? Why not check and display that, before defaulting to the `image_id`?